### DISCUSSION TITLE
Welcome to spaceawesome-feedback Discussions!

### DISCUSSION THREAD

**Discussion by ej2** [FortAwesome/spaceawesome-feedback#2] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place for players to offer feedback and connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

**Comment by stahlmanDesign** [comment]
Well done, game is fun
